yellowbrick/cluster/elbow.py

```diff
@@ -271,8 +322,11 @@ def draw(self):
         Draw the elbow curve for the specified scores and values of K.
         """
         # Plot the silhouette score against k
-        self.ax.plot(self.k_values_, self.k_scores_, marker="D", label="score")
-
+        self.ax.plot(self.k_values_, self.k_scores_, marker="D")
+        if self.locate_elbow and self.elbow_value_!=None:
+            elbow_label = "$elbow\ at\ k={}, score={}$".format(self.elbow_value_, np.round(self.elbow_score_,3))
```

Comment:
Again, not a big deal in this case but my preference is `"{:0.3f}".format(self.elbow_score_)` rather than `np.round` - I think this might be more expected by other developers. It won't be a blocker for this PR though.